app/services/foreman_templates/template_importer.rb

```diff
@@ -69,8 +65,12 @@ def parse_files!
         filename = template.split('/').last
         title    = filename.split('.').first
         name     = metadata['name'] || title
-        name     = [@prefix, name].compact.join
-        next if @filter && !name.match(/#{@filter}/i)
+        name     = auto_prefix(name)
+        if @filter
+          matching = name.match(/#{@filter}/i)
+          matching = !matching if @negate
+          next if !matching
+        end
 
         raise MissingKindError unless metadata['kind']
```

Comment:
The export task isn't writing `kind` for ptables or snippets, so the subsequent import fails. We should probably remove this check now that we're moving to `model` - but we might want to add `model` to the export task so that we always hit line 79 below